plots/laws/thermodynamics/fuel_consumption_and_engine_power.py

```diff
@@ -0,0 +1,41 @@
+from sympy import symbols, Eq
+from sympy.plotting import plot
+from sympy.plotting.plot import MatplotlibBackend
+from symplyphysics import print_expression
+from examples.thermodynamics import energy_from_combustion_of_gasoline as combustion_energy_example
+
+# Note: fuel_consumption = volume_of_fuel / distance
+fuel_consumption = symbols("fuel_consumption")
+
+# Create efficiency factor values from 15% to 75% in increments of 15%
+efficiency_factors = (round(0.15 * (i + 1), 2) for i in range(5))
+
+fuel_consumption_value = (combustion_energy_example.answer.rhs / combustion_energy_example.volume_of_gasoline) ** (-1)
```

Comment:
power of -1 is a weird way to tell, that formula should be inverted. Why not simply exchange numerator and denominator?